addons/point_of_sale/migrations/11.0.1.0.1/openupgrade_analysis_work.txt

```diff
@@ -0,0 +1,111 @@
+---Fields in module 'point_of_sale'---
+point_of_sale / pos.config               / available_pricelist_ids (many2many): NEW relation: product.pricelist
+# Nothing to do
+
+point_of_sale / pos.config               / barcode_scanner (boolean)     : NEW
+# Nothing to do
+
+point_of_sale / pos.config               / customer_facing_display_html (html): NEW
+# Nothing to do
+
+point_of_sale / pos.config               / group_pricelist_item (boolean): NEW
+# Nothing to do
+
+point_of_sale / pos.config               / group_sale_pricelist (boolean): NEW
+# Nothing to do
+
+point_of_sale / pos.config               / iface_customer_facing_display (boolean): NEW
+# Nothing to do
+
+point_of_sale / pos.config               / iface_tax_included (boolean)  : now required, default = function
+point_of_sale / pos.config               / iface_tax_included (boolean)  : selection_keys is now '['subtotal', 'total']' ('False')
+point_of_sale / pos.config               / iface_tax_included (boolean)  : type is now 'selection' ('boolean')
+# Done: update value depend on old boolean value
+
+point_of_sale / pos.config               / iface_tipproduct (boolean)    : NEW
+# Nothing to do
+
+point_of_sale / pos.config               / is_header_or_footer (boolean) : NEW
+# Nothing to do
+
+point_of_sale / pos.config               / is_posbox (boolean)           : NEW
+# Nothing to do
+
+point_of_sale / pos.config               / module_pos_discount (boolean) : NEW
+# Nothing to do
+
+point_of_sale / pos.config               / module_pos_loyalty (boolean)  : NEW
+# Nothing to do
+
+point_of_sale / pos.config               / module_pos_mercury (boolean)  : NEW
+# Nothing to do
+
+point_of_sale / pos.config               / module_pos_reprint (boolean)  : NEW
+# Nothing to do
+
+point_of_sale / pos.config               / module_pos_restaurant (boolean): NEW
+# Nothing to do
+
+point_of_sale / pos.config               / restrict_price_control (boolean): NEW
+# Nothing to do
+
+point_of_sale / pos.config               / sequence_line_id (many2one)   : NEW relation: ir.sequence
+# Nothing to do
+
+point_of_sale / pos.config               / start_category (boolean)      : NEW
+# Nothing to do
+
+point_of_sale / pos.config               / tax_regime (boolean)          : NEW
+# Nothing to do
+
+point_of_sale / pos.config               / tax_regime_selection (boolean): NEW
+# Nothing to do
+
+point_of_sale / pos.config               / use_pricelist (boolean)       : NEW
+# Nothing to do
+
+point_of_sale / pos.session              / rescue (boolean)              : NEW
```

Comment:
Please set rescue = true where name contains 'RESCUE FOR'

Reply:
done @StefanRijnhart !

Reply:
Thanks!